src/main/java/run/halo/app/controller/admin/api/UserController.java

```diff
@@ -99,14 +100,14 @@ public MultiFactorAuthVO generateMFAQrImage(
     @DisableOnCondition
     public MultiFactorAuthVO updateMFAuth(
         @RequestBody @Valid MultiFactorAuthParam multiFactorAuthParam, User user) {
-        if (StrUtil.isNotBlank(user.getMfaKey())
+        if (StringUtils.isNotBlank(user.getMfaKey())
             && MFAType.useMFA(multiFactorAuthParam.getMfaType())) {
             return new MultiFactorAuthVO(MFAType.TFA_TOTP);
-        } else if (StrUtil.isBlank(user.getMfaKey())
+        } else if (StringUtils.isBlank(user.getMfaKey())
             && !MFAType.useMFA(multiFactorAuthParam.getMfaType())) {
             return new MultiFactorAuthVO(MFAType.NONE);
         } else {
-            final String tfaKey = StrUtil.isNotBlank(user.getMfaKey()) ? user.getMfaKey() :
+            final String tfaKey = StringUtils.isNotBlank(user.getMfaKey()) ? user.getMfaKey() :
```

Comment:
I think the variable name `tfaKey` should be `mfaKey`.

Reply:
@JohnNiang ok, done.